src/Compilers/CSharp/Portable/Binder/Binder_Invocation.cs

```diff
@@ -1308,6 +1308,7 @@ private BoundExpression GetDefaultParameterSpecialNoConversion(SyntaxNode syntax
             // Params array is filled in the local rewriter
             var lastIndex = expanded ? ^1 : ^0;
 
+            var arguments = argumentsBuilder.ToImmutableAndFree();
```

Comment:
Only three tests fail due to freeing twice. Does this imply missing freeing elsewhere? I'd expect all tests to fail due to freeing twice. To be more clear, if I removed freeing from here, then there will be some code paths that doesn't free. Is this something I should care about?

Reply:
These are almost always owned by a containing AnalyzedArguments, not owned or freed on their own. There's nothing to check.

---
In reply to: [597457897](https://github.com/dotnet/roslyn/pull/51952#discussion_r597457897) [](ancestors = 597457897)